suites/regressions/generateQueries.js

```diff
@@ -0,0 +1,287 @@
+Feature('Generate queries for performance testing').tag('@regressions');
```

Comment:
ideally, in `suites/regressions` folder, we only have scenarios in which every step will call supporting/detailed/reusable functions stored in `services/regressions`

Reply:
I have PXP-2480 to address this.